yarn.lock

```diff
@@ -6035,11 +6035,6 @@ esbuild@^0.10.2:
   resolved "https://registry.yarnpkg.com/esbuild/-/esbuild-0.10.2.tgz#caa65a8f3096d547d89159918039df6c5c6c90be"
   integrity sha512-/5vsZD7wTJJHC3yNXLUjXNvUDwqwNoIMvFvLd9tcDQ9el5l13pspYm3yufavjIeYvNtAbo+6N/6uoWx9dGA6ug==
 
-esbuild@^0.9.6:
```

Comment:
Don't we need esbuild as a dependency?

Reply:
`esbuild-loader` brings its own `esbuild`.